docs/decisions/2022-07-pet-test-fixtures.md

```diff
@@ -0,0 +1,73 @@
+# Allow "pet" test fixtures
```

Comment:
Per separate discussion, we decided to remove the terms pet and cattle from this decision record in favor of terms like

* cattle: automated, ephemeral or just-in-time infrastructure
* pets: reusable or recyclable infrastructure

Reply:
Done - these terms have been updated. 👍